### ISSUE TITLE
fix(html-spec): add dir as required attribute for bdo element

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#3593]
## Describe the bug

The `required-attr` rule does not report a missing `dir` attribute on `<bdo>` elements. The HTML Living Standard requires the `dir` attribute on `<bdo>`.

> The dir attribute is required on bdo elements.
> — https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-bdo-element

## Code Example

```html
<bdo>text</bdo>
```

## The current behavior

No violation reported.

## The expected behavior

```
The "bdo" element expects the "dir" attribute
```

## Impact

Causes 1 missed error in the nu-html-checker compatibility benchmark (Content Model category).

## Affected package

`@markuplint/html-spec` — bdo element spec data (required attributes)
